app/routines/soft_delete_user.rb

```diff
@@ -0,0 +1,33 @@
+class SoftDeleteUser
+
+  lev_routine
+
+  protected
+
+  def exec(user)
+    return if user.nil?
+
+    # Make sure object up to date, esp before dependent destroy stuff kicks in
+    user.reload
+
+    user.is_deleted = true
+    user.save!
+
+    user.external_ids.destroy_all
+    user.external_uuids.destroy_all
```

Comment:
I didn't even realize we had both. Maybe `external_uuids` can be deleted. I think they just stored the UUID that Tutor used?

Reply:
I'm sure we could have an entire PR dedicated to remove tutor-specific features 😂